src/main.rs

```diff
@@ -18,28 +18,22 @@ fn main() {
     let matches = App::new("caesar-cli")
         .version(crate_version!())
         .about("Demo the caesarlib library with a simple CLI tool.")
-        .arg(Arg::with_name("random")
-             .short("r")
-             .long("random")
-             .conflicts_with("MODE")
-             .help("Choose a random offset and return the offset as the error code."))
         .arg(Arg::with_name("MODE")
-             .help("The action you want to do on the text.")
-             .possible_values(&["encipher", "decipher"])
-             .required_unless("random")
-             .index(1))
-        .arg(Arg::with_name("offset")
-             .short("s")
-             .long("offset")
-             .default_value("13")
-             .help("Set the offset of the caesar code.")
-             .takes_value(true))
+            .help("The action you want to apply on the input text.")
+            .possible_values(&["encipher", "decipher"])
+            .required(true)
+            .index(1))
+        .arg(Arg::with_name("OFFSET")
+            .required_unless("random")
+            .help("Set the offset of the caesar code.")
+            .takes_value(true))
         .arg(Arg::with_name("TEXT")
-             .last(true)
-             .help("The text to process. If not present, will read from stdin."))
+            .required(true)
```

Comment:
I'd like to keep the feature of reading from STDIN, the last needs it to be separated with a `--` like:

``caesarlib --random -- "Foobar"``

or

``cat Cargo.toml | caesarlib encipher -s 24``